pac4j-saml/src/main/java/org/pac4j/saml/credentials/SAML2Credentials.java

```diff
@@ -45,9 +45,22 @@ public SAML2Credentials(final SAMLNameID nameId, final String issuerId,
         this.issuerId = issuerId;
         this.sessionIndex = sessionIndex;
         this.attributes = samlAttributes;
-        this.conditions = new SAMLConditions();
-        this.conditions.setNotBefore(ZonedDateTime.ofInstant(conditions.getNotBefore(), ZoneOffset.UTC));
-        this.conditions.setNotOnOrAfter(ZonedDateTime.ofInstant(conditions.getNotOnOrAfter(), ZoneOffset.UTC));
+        
+		// <Conditions> are optional elements in SAML2 assertions, so if NotBefore or
```

Comment:
Remove inline comments; or switch to javadocs.

Reply:
Inline comments removed. The documentation is in the commit messages/pullrequest and not really relevant to any users, so I'd rather not put it into javadoc.